PROPOSALS.md

```diff
@@ -105,6 +105,13 @@ request feedback on feasibility before investing time and effort in the full pro
     * Unless your branch is particularly big, it's advisable to squash it into
       a single commit. At the moment GitLab does not offer "squash-and-merge"
       option in UI, so this has to be maintained manually.
+6. after the proposal is approved, coordinate with CLC and GHC team:
+    * The proposal needs to be labeled as "approved" and "awaits-merge"
+    * Add a comment to the GHC MR "Approved by CLC in [link to the vote outcome]" and get approval
```

Comment:
Usually an MR was approved by the same member of CLC, who left "Approved by..." comment.